### PR TITLE
Some basic error handling updates and more customization

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

**Review by djzager — APPROVED** [review]
LGTM

File: docs/apb_cli.md [hunk 1]
```diff
@@ -160,6 +160,12 @@ If you are using a namespace other than the default `openshift` namespace to hos
 apb push -o --namespace <namespace>
 ```
 
+This is assuming that the user has the cluster-admin role assigned to the user that is logged in. i.e.:
+```
```
Comment (cfchase) [inline docs/apb_cli.md:164]: do we want to include an OR `oc login -u system:admin`

File: src/apb/cli.py [hunk 2]
```diff
@@ -212,6 +212,20 @@ def subcmd_push_parser(subcmd):
         help=u'Namespace to push to internal OpenShift registry',
         default=u'openshift'
     )
+    subcmd.add_argument(
+        '--registry-service-name',
+        action='store',
+        dest='reg_svc_name',
+        help=u'Namespace where internal registry service lives',
```
Comment (cfchase) [inline src/apb/cli.py]: help copy paste error from namespace.  Should probably be `Service name where...`

File: docs/apb_cli.md (hunk 1 above)
Comment (dymurray) [reply docs/apb_cli.md:164]: Can't use system:admin when we require a token unfortunately.

**Review by cfchase — APPROVED** [review]
ACK

**Pull request merged**